packages/core/upload/admin/src/components/EditAssetDialog/PreviewBox/AssetPreview.jsx

```diff
@@ -21,19 +22,15 @@ export const AssetPreview = forwardRef(({ mime, url, name, ...props }, ref) => {
     return <img ref={ref} src={url} alt={name} {...props} />;
   }
 
-  if (mime.includes(AssetType.Video)) {
+  if (mime.includes(AssetType.Video) || mime.includes(AssetType.Audio)) {
     return (
-      <video controls src={url} ref={ref} {...props}>
-        <track label={name} default kind="captions" srcLang={lang} src="" />
-      </video>
-    );
-  }
-
-  if (mime.includes(AssetType.Audio)) {
-    return (
-      <audio controls src={url} ref={ref} {...props}>
-        {name}
-      </audio>
+      <MuxPlayer
+        src={url}
+        title={name}
+        audio={mime.includes(AssetType.Audio)}
```

Comment:
When passing audio only files I'm seeing an error "Media type  is an unrecognized or unsupported type for src"

Reply:
It seems to be an issue on the Mux Player side, so I will fix it and upstream.